styles.css

```diff
@@ -170,6 +182,116 @@ h2 > a:hover {
   margin-top: -0.8rem;
 }
 
+/* --- DOM Manipulation --- */
+
+.popup {
+  position: fixed;
+  visibility: hidden;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  z-index: 3;
+  background-color: #fff;
+  margin-bottom: 2rem;
+  border-radius: 2px;
+}
+
+.show-popup {
+  display: block;
+  visibility: visible;
+}
+
+.popup-container {
+  height: 100vh;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  background-color: #fff;
+}
```

Comment:
- [ ] The pop-up container is too big, and it's an issue in the mobile and desktop view see the screenshot 👇.

![hhh](https://user-images.githubusercontent.com/61562626/132541134-7e35bc29-d167-49a4-a0c8-0571025870a7.gif)